test/src/main/java/org/apache/accumulo/test/compaction/CompactionPriorityQueueMetricsIT.java

```diff
@@ -407,4 +405,159 @@ public void testQueueMetrics() throws Exception {
     shutdownTailer.set(true);
     thread.join();
   }
+
+  @Test
+  public void newTest() throws Exception {
+    // Metrics collector Thread
+    final LinkedBlockingQueue<TestStatsDSink.Metric> queueMetrics = new LinkedBlockingQueue<>();
```

Comment:
At the beginning of this test or in the `@BeforeEach` function would be good to ensure compactors are stopped. This is  needed if its possible that another test in the class could leave them running.  I think this may be the case because this test class extends SharedMiniClusterBase.

Reply:
Addressed in bd4d08e